src/main/rules/GCI97/GCI97.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
in this PR, there are some lacks : you should put the modification of

- RULES.md to add inside this rule in the array
- update CHANGELOG.md